### PR TITLE
Fix typo in Javadoc

### PR DESCRIPTION
**Pull request by dhardtke**
Found a small typo 🥇

### CODE REVIEW

**Comment by dhardtke** [comment]
It should of course say Javadoc instead of documentation - it's late, I should probably go to bed

File: core/src/main/java/org/jdbi/v3/core/statement/PreparedBatch.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
 import java.util.Map;
 import java.util.NoSuchElementException;
 
+import edu.umd.cs.findbugs.annotations.SuppressWarnings;
```
Comment (TheRealMarnes) [inline core/src/main/java/org/jdbi/v3/core/statement/PreparedBatch.java]: This import needs to be removed, it changes the SuppressWarnings down below from the java lang one to the findbugs one.

**Comment by TheRealMarnes** [comment]
Remove the stowaway import and we'll 🚢

Thanks!

File: core/src/main/java/org/jdbi/v3/core/statement/PreparedBatch.java (hunk 1 above)
Comment (dhardtke) [reply core/src/main/java/org/jdbi/v3/core/statement/PreparedBatch.java]: Got it. Weird, dunno how it got there in the first place.

Comment (TheRealMarnes) [reply core/src/main/java/org/jdbi/v3/core/statement/PreparedBatch.java]: probably IDE auto-import

**Pull request merged**